ui/i18n/datepicker-ur.js

```diff
@@ -0,0 +1,47 @@
+/* Abdullah initialisation for the jQuery UI date picker plugin. */
+/* For Pakistani Users Only */
+/* Written by Abdullah <abdullahmzm@gmail.com> */
+/* Twitter Handle @Software_abi */
+(function (factory) {
+    if (typeof define === "function" && define.amd) {
+
+        // AMD. Register as an anonymous module.
+        define(["../widgets/datepicker"], factory);
+    } else {
+
+        // Browser globals
+        factory(jQuery.datepicker);
+    }
+}(function (datepicker) {
+
+    datepicker.regional.ur = {
+        closeText: "بند کریں",
+        prevText: "&#x3C;پچھلا",
+        nextText: "اگلا&#x3E;",
+        currentText: "آج",
+        monthNames: ["جنوری", "فروری", "مارچ", "اپریل", "مئی", "جون",
+            "جولائی", "اگست", "ستمبر", "اکتوبر", "نومبر", "دسمبر"],
+        monthNamesShort: ["1", "2", "3", "4", "5", "6",
```

Comment:
No short names in Urdu?

Reply:
Yes, there are no short names for months in Urdu.